pkg/apis/core/v1alpha1/conversions.go

```diff
@@ -25,6 +25,20 @@ import (
 )
 
 func addConversionFuncs(scheme *runtime.Scheme) error {
+	if err := scheme.AddFieldLabelConversionFunc(
+		SchemeGroupVersion.WithKind("ControllerInstallation"),
+		func(label, value string) (string, string, error) {
+			switch label {
+			case "metadata.name", "metadata.namespace", core.RegistrationRefName, core.SeedRefName:
```

Comment:
`metadata.namespace` is not needed, right?
Should we leave it out then?

```suggestion
			case "metadata.name", core.RegistrationRefName, core.SeedRefName:
```

Reply:
Nice catch, thank you!